src/components/TextareaField/TextareaField.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { generateSnapshots } from '@chanzuckerberg/story-utils';
+import { screen } from '@storybook/testing-library';
+
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import React from 'react';
+import { TextareaField } from './TextareaField';
+import * as stories from './TextareaField.stories';
+
+describe('<TextareaField />', () => {
+  generateSnapshots(stories);
+
+  it('can handle custom events when there is content', async () => {
+    const onChangeFn = jest.fn(() => console.log('test'));
+    const user = userEvent.setup();
+    render(
+      <TextareaField aria-label="test" onChange={() => onChangeFn()}>
+        some content
+      </TextareaField>,
+    );
+
+    const field = screen.getByRole('textbox');
+
+    expect(field).not.toHaveFocus();
+
+    await userEvent.tab();
+
+    expect(field).toHaveFocus();
+
+    await user.keyboard('{arrowdown}{delete}');
+
+    expect(onChangeFn).toHaveBeenCalled();
+  });
+
+  it('will not fire a custom event when there is no content', async () => {
```

Comment:
By default, no change event is triggered if an empty field is edited. This test ensures we don't break that rule at some point in the future.